x/emissions/module/rewards_calc.go

```diff
@@ -182,14 +182,14 @@ func MaskWeightsIfInsufficientLiveness(
 			if err != nil {
 				return nil, err
 			}
-			maxPossibleInferencesInRewardEpoch := uint64(rewardCadence) / topic.EpochLength
+			maxPossibleInferencesInRewardEpoch := rewardCadence / topic.EpochLength
```

Comment:
this entire file is deleted in the branch you are trying to merge into so probably this goes away when you go to merge